themes/hadakewa/partials/content.php

```diff
@@ -74,9 +119,9 @@
 								<div class="img">
 									<?php if ($data['gambar']!=''): ?>
 										<?php if (is_file(LOKASI_FOTO_ARTIKEL."kecil_".$data['gambar'])): ?>
-											<img src="<?= AmbilFotoArtikel($data['gambar'],'kecil') ?>" alt="<?= $data["judul"] ?>"/>
+											<img src="<?= AmbilFotoArtikel($data['gambar'],'kecil') ?>" alt="<?= $data["judul"] ?>" width="300" style="float:left; margin:0 8px 4px 0;"/>
 											<?php else: ?>
-												<img src="<?= base_url('assets/images/404-image-not-found.jpg') ?>" alt="<?= $data["judul"] ?>" />
+												<img src="<?= base_url('assets/images/404-image-not-found.jpg') ?>" alt="<?= $data["judul"] ?>" width="300" style="float:left; margin:0 8px 4px 0;"/>
 											<?php endif;?>
```

Comment:
Stylenya bisa di defnisikan di awal agar tdk berulang dan inline.

Bisa pakai (is_file(LOKASI_FOTO_ARTIKEL."kecil_".$data['gambar'])) ? AmbilFotoArtikel($data['gambar'],'kecil') : base_url('assets/images/404-image-not-found.jpg');

Reply:
Sebagai info tambahan, style yg dimasukkan inline atau di file view tidak bisa ditimpa/diubah menggunakan css di folder desa. Sejauh mungkin, masukkan style di `assets/front/css/first.css`, sehingga bisa diubah desa mengikuti petunjuk https://github.com/OpenSID/OpenSID/wiki/Mengubah-Penampilan-Web